.github/workflows/docker.yml

```diff
@@ -18,68 +22,78 @@
   group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
   cancel-in-progress: true
 jobs:
-  docker:
+  heighliner-docker:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+
+      - uses: docker/setup-buildx-action@v3
+
+      - uses: docker/setup-qemu-action@v2
+
+      - uses: docker/metadata-action@v5
+        id: meta
+        with:
+          images: ${{ env.DOCKER_IMAGE }}
+          flavor: |
+            prefix=heighliner-,onlatest=true
+          tags: |
+            type=edge
+            type=ref,enable=${{ github.event_name == 'pull_request' }},event=branch
+            type=ref,event=tag
+
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+        # - uses: strangelove-ventures/heighliner-build-action@v1.0.1
+      - uses: strangelove-ventures/heighliner-build-action@main
         with:
-          fetch-depth: 0
+          heighliner-tag: v1.5.5
+          chain: provenance
+          local: true
+          tag: ${{ steps.meta.outputs.version }}
+          registry: provenanceio
+          additional-args: --alpine-version=3.18
+          skip: ${{ github.event_name == 'pull_request' }}
```

Comment:
Stop making this suggestion. I keep resolving it because you're so very wrong about this. You seem to know nothing about the `heighliner-tag` field and are just spouting things to hear your own voice.

Reply:
It's a version value that is not located or used anywhere else in our code base. We know what value to use by examining releases made by the heighliner team (which isn't related to us at all).